src/App.tsx

```diff
@@ -1,14 +1,55 @@
-/* eslint-disable max-len */
-import React from 'react';
+import {
+  FC,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [todoId, setTodoId] = useState(0);
```

Comment:
use on default value null if something doesn’t have
```suggestion
  const [todoId, setTodoId] = useState(null);
```